0-common/schema-ddl/project/BuildSettings.scala

```diff
@@ -25,11 +25,10 @@ object BuildSettings {
     organization          :=  "com.snowplowanalytics",
     version               :=  "0.6.0",
     description           :=  "Set of Abstract Syntax Trees for various DDL and Schema formats",
-    scalaVersion          :=  "2.10.6",
-    crossScalaVersions    :=  Seq("2.10.6", "2.11.8"),
+    scalaVersion          :=  "2.12.4",
+    crossScalaVersions    :=  Seq("2.11.8", "2.12.4"),
```

Comment:
Let's also silently bump 2.11 to 2.11.12 here (don't want to create another super-minor ticket)